R-package/tests/testthat/test_dataset.R

```diff
@@ -205,3 +205,51 @@ test_that("Dataset$update_params() works correctly for recognized Dataset parame
     expect_identical(new_params[[param_name]], updated_params[[param_name]])
   }
 })
+
+test_that("lgb.Dataset: should be able to run lgb.train() immediately after using lgb.Dataset() on a file", {
+  dtest <- lgb.Dataset(data = test_data
+                       , label = test_label)
```

Comment:
```suggestion
  dtest <- lgb.Dataset(
    data = test_data
    , label = test_label
  )
```

Can you please use this style everywhere? With leading commas, and a wrapping final `)`?

Reply:
Done I think!